Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -158,6 +158,11 @@ private object MapActivityToLiteDb(Activity activity, Context context, eExecuted
             }
             activity.Acts.ToList().ForEach(action => this.MapActionToLiteDb((GingerCore.Actions.Act)action, context, executedFrom));
             AR.ActionsColl.AddRange(liteDbActionList);
+            //counts
+            AR.ChildExecutableItemsCount.Add("Action", activity.Acts.Count);
+            AR.ChildExecutedItemsCount.Add("Action", (activity.Acts.Count - activity.Acts.Count(x => x.Status == eRunStatus.Pending || x.Status == eRunStatus.Skipped || x.Status == eRunStatus.Blocked)));
+            AR.ChildPassedItemsCount.Add("Action", activity.Acts.Count(x => x.Status == eRunStatus.Passed));
```

Comment:
for Dictionaries Keys do not use static String "Activity"/"Actions"- try to avoid using static strings in code as much as possible.
instead use Enum called "eExecutionStatisticsCountBy" with values "Activities" and "Actions".
use this enum for the Dics Keys, you can do "eExecutionStatisticsCountBy.Activities.ToString()".
use this enum in all relevant classes/pages so all will be based on same enum